lib/puppet/resource_api.rb

```diff
@@ -210,7 +210,11 @@ def feature_support?(feature)
 
           case options[:type]
           when 'Enum[present, absent]'
-            Puppet::ResourceApi.def_newvalues(self, param_or_property, :absent, :present)
+            Puppet::ResourceApi.def_newvalues(self, param_or_property, 'absent', 'present')
```

Comment:
jeez 😞

Reply:
the real bummer is the next line. this here is only cosmetics